docs/topics/compiler-reference.md

```diff
@@ -234,6 +234,15 @@ Allows using the `returnsNotNull()` function in contracts to assume a non-null r
 
 Allows using the `holdsIn` keyword in contracts to assume that a boolean condition is `true` inside a lambda.
 
+### -Xreturn-value-checker
+<primary-label ref="experimental-general"/>
+
+Configure how the compiler reports ignored results:
```

Comment:
Should this line contain a link to the new documentation page? Otherwise users won't be able to learn more about feature.